src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CallVariantsFromAlignedContigsSpark.java

```diff
@@ -0,0 +1,305 @@
+                .start(start)
+                .stop(end)
+                .id(getInversionId(breakpointAllele))
+                .alleles(vcAlleles)
+                .attribute(VCFConstants.END_KEY, end)
+                .attribute(GATKSVVCFHeaderLines.SVTYPE, GATKSVVCFHeaderLines.SVTYPES.INV.toString())
+                .attribute(GATKSVVCFHeaderLines.SVLEN, end - start)
+                .attribute(GATKSVVCFHeaderLines.TOTAL_MAPPINGS, numAssembledBreakpoints)
+                .attribute(GATKSVVCFHeaderLines.HQ_MAPPINGS, highMqMappings)
+                .attribute(GATKSVVCFHeaderLines.MAPPING_QUALITIES, mqs.stream().map(String::valueOf).collect(Collectors.joining(",")))
+                .attribute(GATKSVVCFHeaderLines.ALIGN_LENGTHS, alignLengths.stream().map(String::valueOf).collect(Collectors.joining(",")))
+                .attribute(GATKSVVCFHeaderLines.MAX_ALIGN_LENGTH, maxAlignLength)
+                .attribute(GATKSVVCFHeaderLines.BREAKPOINT_IDS, breakpointIds.stream().collect(Collectors.joining(",")))
+                .attribute(GATKSVVCFHeaderLines.CONTIG_IDS, assembledContigIds.stream().map(s -> s.replace(" ", "_")).collect(Collectors.joining(",")));
+
+        if (breakpointAllele.insertedSequence.length() > 0) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.INSERTED_SEQUENCE, breakpointAllele.insertedSequence);
+        }
+
+        if (breakpointAllele.homology.length() > 0) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.HOMOLOGY, breakpointAllele.homology);
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.HOMOLOGY_LENGTH, breakpointAllele.homology.length());
+        }
+
+        if (is5To3Inversion(breakpointAllele)) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.INV_5_TO_3, "");
+        }
+
+        if (is3To5Inversion(breakpointAllele)) {
+            vcBuilder = vcBuilder.attribute(GATKSVVCFHeaderLines.INV_3_TO_5, "");
+        }
+
+
+        return vcBuilder.make();
+    }
+
+    private static boolean is3To5Inversion(final BreakpointAllele breakpointAllele) {
+        return ! breakpointAllele.fiveToThree && breakpointAllele.threeToFive;
+    }
+
+    private static boolean is5To3Inversion(final BreakpointAllele breakpointAllele) {
+        return breakpointAllele.fiveToThree && ! breakpointAllele.threeToFive;
+    }
+
+    private static String getInversionId(final BreakpointAllele breakpointAllele) {
+        final String invType;
+        if (is5To3Inversion(breakpointAllele)) {
+            invType = "INV_5_TO_3";
+        } else if (is3To5Inversion(breakpointAllele)) {
+            invType = "INV_3_TO_5";
+        } else {
+            // I don't think this should happen
+            invType = "INV_UNKNOWN";
+        }
+
+        return invType + "_" + breakpointAllele.leftAlignedLeftBreakpoint.getContig() + "_" + breakpointAllele.leftAlignedLeftBreakpoint.getStart() + "_" + breakpointAllele.leftAlignedRightBreakpoint.getStart();
+    }
```

Comment:
I propose the following change to the code:

1. remove functions `is5To3Inversion` and `is5To3Inversion`;
2. To make things work, add the following to class `BreakpointAllele`
    ```java
public enum InversionType{
        INV_3_TO_5, INV_5_TO_3, INV_UNKNOWN
    }

    public InversionType getInversionType(){
        if(!fiveToThree && threeToFive){
            return InversionType.INV_3_TO_5;
        }else if(fiveToThree && !threeToFive){
            return InversionType.INV_5_TO_3;
        }
        return InversionType.INV_UNKNOWN;
    }
    ```
3. To get string representation of the inversion type, it would now become something like `breakpoingAllele.getInversionType().name()`.